nss-day-cohort-51/nss-day-cohort-51.github.io
# mw-feat-CreateTechStackComponent-3: Added the code that displays the …

…tech stacks lists for all 3 sections; Tech Stack Button is created and works to display each stack object.
#3 #7 
# Description
Completed both Tech Stack component and Tech Stack Card issues.

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions
- Pull down branch and run web page in browser
- Check that each of the 3 sections display a list of tech stack items 

# Checklist:
